fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1206,9 +1189,12 @@ def _init_param_attributes(self, p: Parameter) -> None:
             # shard in pinned memory so that we can do a non-blocking transfer.
```

Comment:
maybe move the comment about the pinning to where we actually pin the memory.

Reply:
Moved around the 2 comments about pinned memory in my next commit so hopefully it's a bit easier to understand.